staging/src/k8s.io/kube-aggregator/pkg/controllers/openapiv3/aggregator/aggregator.go

```diff
@@ -268,6 +269,24 @@ func (s *specProxier) handleGroupVersion(w http.ResponseWriter, r *http.Request)
 
 // Register registers the OpenAPI V3 Discovery and GroupVersion handlers
 func (s *specProxier) register(handler common.PathHandlerByGroupVersion) {
-	handler.Handle("/openapi/v3", http.HandlerFunc(s.handleDiscovery))
-	handler.HandlePrefix("/openapi/v3/", http.HandlerFunc(s.handleGroupVersion))
+	handler.Handle("/openapi/v3", metrics.InstrumentHandlerFunc("GET",
+		/* group = */ "",
+		/* version = */ "",
+		/* resource = */ "",
+		/* subresource = */ "openapi/v3",
+		/* scope = */ "",
+		/* component = */ "",
+		/* deprecated */ false,
+		/* removedRelease */ "",
+		http.HandlerFunc(s.handleDiscovery)))
+	handler.HandlePrefix("/openapi/v3/", metrics.InstrumentHandlerFunc("GET",
+		/* group = */ "",
+		/* version = */ "",
+		/* resource = */ "",
+		/* subresource = */ "openapi/v3/",
```

Comment:
This slash is intentional, would like to separate requests to OpenAPI V3 discovery (root document), and requests to actual OpenAPI documents.